pnpm-lock.yaml

```diff
@@ -29834,25 +29834,25 @@ snapshots:
   '@babel/plugin-transform-classes@7.23.5(@babel/core@7.23.5)':
```

Comment:
Are these package version changes intended?

Reply:
yeah, looks risky, worth double checking

Reply:
No. I have no idea how they keep slipping into my PRs. Has anyone else noticed the same? 🤔

Reply:
yeah, probably that is happening when running `pnpm i` - it works like `composer update`
I use now ` pnpm install --frozen-lockfile` - it won't update only install from lockfile!

Reply:
Great, I'll try that. Thank you!

Reply:
Resolved!